external-import/recorded-future/src/rflib/pyrf.py

```diff
@@ -513,7 +528,7 @@ def get_prioritedrule_ids(self, limit=100):
                 data = response.json()
 
                 # If the response doesn't contain data, log the error
-                if not data or data.get("data"):
+                if not data or not data.get("data"):
```

Comment:
```suggestion
                if data.get("data"):
```
Should be sufficient as `data = response.json() ` is the above line.